lib/packer/provisioners/puppet/masterless.rb

```diff
@@ -0,0 +1,64 @@
+# Encoding: utf-8
+# Copyright 2015 Fraser Cobb
```

Comment:
The project license trumps the file comments. I should probably just remove these across all the files in the project...